drivers/flash/soc_flash_nrf.c

```diff
@@ -312,8 +319,8 @@ static void time_slot_callback_work(u32_t ticks_at_expire, u32_t remainder,
 	}
 }
 
-static void time_slot_callback_helper(u32_t ticks_at_expire, u32_t remainder,
-		u16_t lazy, void *context)
+static void time_slot_callback_abort(u32_t ticks_at_expire, u32_t remainder,
+				     u16_t lazy, void *context)
```

Comment:
Spaces.

Reply:
4 tabs and 5 spaces are correct indentation to align with the first parameter definition.

Reply:
Ah, sorry. My bad.